vignettes/quick-start.Rmd

```diff
@@ -0,0 +1,161 @@
+---
+title: "Quick Start"
+author: "R Validation Hub"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Quick Start}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+library(riskmetric)
+library(dplyr)
+library(tibble)
+
+options(repos = "https://cran.rstudio.com")
+
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+Whether we're looking for a robust choice among a set of similar packages,
+require strong development and maintainence practices for package reliability,
+or need to prove the validity of packages in a regulated industry, we will
+follow the same sequence of steps.
+
+1. **Find a suitable reference for the package**  
+   this can be any source of package metadata, such as a
+    * package repository webportal from which we can scrape metadata
+    * built and installed package in a local library
+    * directory containing package source code 
+1. **Assess package under validation criteria**  
```

Comment:
I assume 2. 3. 4. shall be used in each bold item

Reply:
In markdown, multiple `1.`'s will automatically be incremented - it makes it easier to add elements in the middle of a list without having to edit all the numbers after it. Here's an example (all are numbered `1.`.

1. one
1. two
1. three